src/main/java/com/solvd/pages/android/components/ProductListItemComponent.java

```diff
@@ -1,13 +1,14 @@
 package com.solvd.pages.android.components;
 
 import com.zebrunner.carina.utils.android.AndroidService;
+import com.zebrunner.carina.utils.mobile.IMobileUtils;
 import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
 import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
 import io.appium.java_client.pagefactory.AndroidFindBy;
 import org.openqa.selenium.SearchContext;
 import org.openqa.selenium.WebDriver;
 
-public class ProductListItemComponent extends AbstractUIObject{
+public class ProductListItemComponent extends AbstractUIObject implements IMobileUtils {
```

Comment:
create Abstract component and implements IMobileUtils  as I showed you in my example